packages/react-sdk/src/components/image.stories.tsx

```diff
@@ -1,18 +1,145 @@
+  docs: {
+    description: {
+      story: "Load images depending on image width and device per pixel ratio.",
+    },
+  },
+};
+
+FixedWidthImage.args = {
+  src: "https://assets-dev.webstudio.is/0beWEKDTrKqaZkh5bdN0peW0oOrcXN_1663019526493.jpg",
+  width: 400,
+  height: 400,
+};
+
+export const FixedWidthImageCover = ImageTemplate.bind({});
+
+FixedWidthImageCover.parameters = {
+  docs: {
+    description: {
+      story:
+        "Preserve ratio using object-fit: cover. Load images depending on image width and device per pixel ratio.",
+    },
+  },
+};
+
+FixedWidthImageCover.args = {
+  src: "https://assets-dev.webstudio.is/0beWEKDTrKqaZkh5bdN0peW0oOrcXN_1663019526493.jpg",
+  width: 400,
+  height: 400,
+  css: { objectFit: "cover" },
+};
+
+export const UnknownWidthImage = ImageTemplate.bind({});
+
+UnknownWidthImage.parameters = {
+  docs: {
+    description: {
+      story: `Load images depending on the viewport width.`,
+    },
+  },
+};
+
+UnknownWidthImage.args = {
+  src: "https://assets-dev.webstudio.is/0beWEKDTrKqaZkh5bdN0peW0oOrcXN_1663019526493.jpg",
+};
+
+FixedWidthImageCover.args = {
+  src: "https://assets-dev.webstudio.is/0beWEKDTrKqaZkh5bdN0peW0oOrcXN_1663019526493.jpg",
+  width: 400,
+  height: 400,
+  css: { objectFit: "cover" },
+};
+
+export const AspectRatioImage = ImageTemplate.bind({});
+
+AspectRatioImage.parameters = {
+  docs: {
+    description: {
+      story: `Fit width of the parent container, has own aspect-ratio and object-fit=cover. Load images depending on the viewport width.`,
+    },
+  },
+};
+
+AspectRatioImage.decorators = [
+  (Story) => <div style={{ maxWidth: "400px" }}>{<Story />}</div>,
+];
+
+AspectRatioImage.args = {
+  src: "https://assets-dev.webstudio.is/0beWEKDTrKqaZkh5bdN0peW0oOrcXN_1663019526493.jpg",
+  css: { aspectRatio: "3/4", objectFit: "cover" },
+};
+
+export const FillParentImage = ImageTemplate.bind({});
+
+FillParentImage.parameters = {
+  docs: {
+    description: {
+      story: `Fill width and height of the relative parent container, object-fit=cover. Load images depending on the viewport width.`,
+    },
+  },
+};
+
+FillParentImage.decorators = [
+  (Story) => (
+    <div
+      style={{ maxWidth: "400px", aspectRatio: "2/1", position: "relative" }}
+    >
+      {<Story />}
+    </div>
+  ),
+];
+
+FillParentImage.args = {
+  src: "https://assets-dev.webstudio.is/0beWEKDTrKqaZkh5bdN0peW0oOrcXN_1663019526493.jpg",
+  css: {
+    position: "absolute",
+    left: 0,
+    right: 0,
+    top: 0,
+    bottom: 0,
+    objectFit: "cover",
+  },
+};
+
+export const HeroImage = ImageTemplate.bind({});
+
+HeroImage.parameters = {
+  docs: {
+    description: {
+      story: `"sizes" attribute explicitly equal to 100vw allowing to skip the default behavior. See DEFAULT_SIZES in the Image component. Load images depending on the viewport width.`,
+    },
+  },
+};
+
+HeroImage.args = {
+  src: "https://assets-dev.webstudio.is/0beWEKDTrKqaZkh5bdN0peW0oOrcXN_1663019526493.jpg",
```

Comment:
lets use some local dummy image instead

Reply:
Done, as discussed `USE_CLOUDFLARE_IMAGE_TRANSFORM` added, and organisation logo used.